.github/workflows/python_ci.yaml

```diff
@@ -0,0 +1,67 @@
+name: Python CI
+
+on:
+  pull_request:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+env:
+  MANIFOLD_API_KEY: ${{ secrets.MANIFOLD_API_KEY }}
```

Comment:
Most of these can be removed, or can't they?

Reply:
removed